terraform/scripts/update_geoip.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
```

Comment:
I would like to see a brief explanatory comment summarizing what this script does, similar to [this](https://github.com/cisagov/cool-assessment-terraform/blob/develop/cloud-init/nessus-setup.sh#L3-L15).

Reply:
@dav3r Resurrecting this ancient thing to close it out. Please see [86847d1](https://github.com/cisagov/cyhy_amis/pull/362/commits/86847d1cd7fdad9d8b92c4792c5599bf950a3aae) and let me know if that suffices.

Reply:
Yes, that is above and beyond what I was looking for - very comprehensive. 👍